pages/release-process.md

```diff
@@ -1,111 +1,145 @@
-[documentation]: http://central.sonatype.org/pages/apache-maven.html
-[sonatype account]: https://issues.sonatype.org/
-[Release the artifacts]: http://central.sonatype.org/pages/releasing-the-deployment.html
+---
+layout: page
+title: Release Process
+permalink: /release-process/
+---
+
+### Initial Setup
+
+Before you can release Fluo or Fluo Recipes, you will need a GPG key. For information on generating
+a key look at this [ASF GPG page](https://www.apache.org/dev/openpgp.html).  After generating a key,
+add it to the [KEYS] file.  The [KEYS] files contains instructions for adding to itself.  Use the 
+following command to checkout the svn repository that contains the KEYS files.  Updates to this
+repository will eventually sync to the website.
+
+```bash
+svn co https://dist.apache.org/repos/dist/release/incubator/fluo/
+```
+
+The maven release plugin will need credentials to stage artifacts.  You can provide the credentials
+by adding the following to your `~/.m2/settings.xml` file.  Maven offers documentation about [securing
+your credentials](https://maven.apache.org/guides/mini/guide-encryption.html).
+
+```xml
+<servers>
+  <server>
+    <!-- Project using the Apache parent pom use following ID -->
+    <id>apache.releases.https</id>
+    <username>your-apache-id</username>
+    <password>your-apache-password</password>
+  </server>
+</servers>
+```
+
+### Release Fluo
+
+Before starting the release process below, the following tasks should be complete:
+
+ * Ensure the NOTICE file has the correct year.
+ * Create release notes for project website using GitHub issues.
+ * Perform testing and document results.
+ * Start a gpg-agent to cache your gpg key to avoid entering your passphrase multiple times.  How
+   you start this depends on your environment.  The following command works in some environments. 
+   Ensure gpg-agent is configured with a sufficiently long timeout so that the cached passphrase 
+   do not expire during the build.
+
+   ```shell
+   gpg-agent --daemon --use-standard-socket
+   ```
+
+Next, repeat the steps below until a good release candidate (RC) is found.  The script in
+`contrib/create-release-candidate.sh` automates this process.  However there is no guarantee that it
+works correctly.  Before using the script ensure you understand the process and inspect the script.
+In the following steps `RCV` is short for release candidate version.  For the case where you want to
+make an initial release candidate available for evaluation, but nor for voting, consider using `0`
+for `RCV`.
+
+ 1. Branch master (or the current snapshot) and call the branch `<releaseVersion>-rc<RCV>-next`
+
+ 2. Prepare the release which will verify that all tests pass: `mvn release:prepare`
+
+ 3. Perform the release: `mvn release:perform`
+    * This step will create a staging repository viewable only to you when login to https://repository.apache.org
```

Comment:
I decided to drop the whole bit.  Even it we make it correct, I feel its not very helpful/useful info.